src/components/WalletBtn/WalletBtn.module.css

```diff
@@ -1,99 +1,60 @@
-.dropdown_list {
-  top: unset;
-  bottom: 150%;
+.btn_connect, .wallet_btn_dropdown {
+  height: 40px;
+  text-transform: uppercase;
+  border: none;
+  line-height: 1rem;
+  border-radius: 2.5rem;
+  cursor: pointer;
 }
 
 .btn_connect {
-  background-color: transparent;
+  width: 154px;
+  background-color: var(--green-300);
+  transition: all 0.2s ease-in-out;
+  color: var(--dark-green-700);
+  cursor: pointer;
 }
 
-.authenticating {
-  font-size: 2rem;
-  width: 100%;
-  height: 100%;
+.btn_connect:hover {
+  background-color: var(--green-500);
+  box-shadow: var(--green-box-shadow);
+}
+
+.wallet_btn_dropdown, .dropdown_icons, .dropdown_wrapper   {
   display: flex;
-  justify-content: center;
   align-items: center;
 }
 
 .wallet_btn_dropdown {
-  width: 3.5rem;
-  height: 3.5rem;
-  border-radius: 50vh;
-  border: none;
-  padding: 0.5rem;
-  cursor: pointer;
-  animation-name: blinks;
-  animation-duration: 1s;
-  animation-timing-function: cubic-bezier(0.075, 0.82, 0.165, 1);
-  animation-iteration-count: infinite;
-  animation-fill-mode: backwards;
-  box-shadow: 0px 0px 50px -10px #7fe89d;
+  justify-content: space-between;
+  width: 159px;
+  height: 32px;
+  padding-inline: 0.5rem;
+  background: var(--white-40);
+  border: 1px solid var(--primary-700);
+  border-radius: 6.25rem;
+  color: var(--grey-01);
 }
 
-.wallet_btn_dropdown img {
-  width: 100%;
-  height: 100%;
-  object-fit: contain;
+.dropdown_icons {
+  gap: 0.55rem;
 }
 
-.wallet_btn_dropdown_disconnected {
-  animation-name: blinks_disconnected;
-  box-shadow: 0px 0px 50px -10px #e73030;
-}
-.wallet_btn_dropdown_connecting {
-  animation-name: blinks_connecting;
-  box-shadow: 0px 0px 50px -10px #e73030;
+.avatar {
+  width: 1.5rem;
+  height: 1.5rem;
 }
 
-@keyframes blinks {
-  0% {
-    box-shadow: 0px 0px 10px -10px #7fe89d;
-  }
-  25% {
-    box-shadow: 0px 0px 20px -10px #7fe89d;
-  }
-  50% {
-    box-shadow: 0px 0px 30px -10px #7fe89d;
-  }
-  75% {
-    box-shadow: 0px 0px 40px -10px #7fe89d;
-  }
-  100% {
-    box-shadow: 0px 0px 50px -10px #7fe89d;
-  }
-}
-@keyframes blinks_disconnected {
-  0% {
-    box-shadow: 0px 0px 10px -10px #e73030;
-  }
-  25% {
-    box-shadow: 0px 0px 20px -10px #e73030;
-  }
-  50% {
-    box-shadow: 0px 0px 30px -10px #e73030;
-  }
-  75% {
-    box-shadow: 0px 0px 40px -10px #e73030;
-  }
-  100% {
-    box-shadow: 0px 0px 50px -10px #e73030;
-  }
+.dropdown_wrapper {
+  flex: 1;
+  display: flex!important;
```

Comment:
is there a way to do this without !important?

Reply:
It's overriding a custom style but will look into it.